netbout-web/src/main/java/com/netbout/email/BoutInviteMail.java

```diff
@@ -89,7 +89,7 @@
 
     /**
      * Send an email with register link which is like.
-     * http://www.netbout.com/b/<bout_number>?invite=<invite-key>
+     * http://www.netbout.com/b/<bout_number/>?invite=<invite-key/>
```

Comment:
@mkordas Even if I use pre 
```
 /**
     * Send an email with register link which is like.
     * <pre>
     * http://www.netbout.com/b/<bout_number>?invite=<invite-key>
     * </pre>
     * @param email Email
     * @param urn Urn
     * @param bout Bout
     * @throws IOException if fails
     */
```
It is a violation :  Javadoc comment at column 54 has parse error. Missed HTML close tag 'invite-key'. Sometimes it means that close tag missed for one of previous tags. (AtclauseOrderCheck)

As I can see this comment only serves to show how the link will be formated so I could write something like  : 
```
http://www.netbout.com/b/{bout_number}?invite={invite-key}
```
Tell me if I can use this approach.